Sasquatch/Sasquatch/Base.lproj/LaunchScreen.storyboard

```diff
@@ -1,11 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13529" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="D7o-BS-rls">
-    <device id="retina5_9" orientation="portrait">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="12121" systemVersion="17G3025" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" launchScreen="YES" useTraitCollections="YES" colorMatched="YES" initialViewController="D7o-BS-rls">
+    <device id="retina4_7" orientation="portrait">
```

Comment:
Why are these versions being decreased?

Reply:
We opened the storyboard in Xcode 8.3.3 which causes those changes as Xcode 8 does not know about `retina5_9`.